lib/cdo/pardot.rb

```diff
@@ -272,6 +272,13 @@ def self.apply_special_fields(src, dest)
       dest[:is_do_not_email] = true
     end
 
+    # The custom Pardot db_Opt_In field has type "Dropdown" with permitted values "Yes" or "No".
+    # Explicitly check for the source opt_in field to be true or false, because nil is 'falsey'
+    # and it's a little misleading to set a value for db_Opt_In in Pardot when there's no information either
+    # way from the user in our database.
+    dest[:db_Opt_In] = 'Yes' if src[:opt_in] == true
```

Comment:
For now, I suggest we avoid setting `db_Opt_In` in Pardot if we the value is `nil` in our database, just to avoid overwriting information in Pardot that we haven't yet initialized in our database.